### ISSUE TITLE
Add shadow effect and custom corner radius

### ISSUE THREAD

**Issue by unofficialdev** [AvaloniaUtils/DialogHost.Avalonia#3]
consider add shadow effect and custom corner radius.
The shadow effect makes the dialog appear taller than the owner content.

**Comment by SKProCH** [comment]
You can customize DialogContentTemplate by yourself, here's default template: 
```
<Style Selector="dialogHost|DialogHost">
    <Setter Property="PopupTemplate">
        <ControlTemplate>
            <VisualLayerManager IsPopup="True" Name="PART_DialogPopupRoot">
                <Border Name="PART_ContentBackground" CornerRadius="2">
                    <ContentPresenter Name="PART_ContentPresenter"
                                      Background="{TemplateBinding Background}"
                                      ContentTemplate="{TemplateBinding ContentTemplate}"
                                      Content="{TemplateBinding Content}"
                                      Padding="{TemplateBinding Padding}" />
                </Border>
            </VisualLayerManager>
        </ControlTemplate>
    </Setter>
</Style>
```

**Comment by SKProCH** [comment]
Hello, @unofficialdev 
I implemented it in new 0.3.0 version: 
- Add shadow for default DialogHost popup style
- Add controllable CornerRadius via DialogHostStyle.CornerRadius attached property

**Comment by SKProCH** [comment]
Released [v0.3.0](https://www.nuget.org/packages/DialogHost.Avalonia/0.3.0)
